packages/compass-sidebar/src/components/legacy/sidebar-databases-navigation.tsx

```diff
@@ -168,7 +168,9 @@ const onNamespaceAction = (
         emit('open-drop-collection', connectionId, ns);
         return;
       case 'create-collection':
-        emit('open-create-collection', ns);
+        emit('open-create-collection', ns, {
+          connectionId,
+        });
```

Comment:
this might be my OCD, but is there a reason not to follow the same pattern as the actions above?

Reply:
no actually you're right - we diverged already from the agreed upon pattern proposed by (ConnectionScopedAppRegistry) so in this PR I am following that and currently working on aligning the rest of the events to the same pattern.